### PR TITLE
Update Scala to 2.12.0.

### PR DESCRIPTION
**Pull request by odd**

### CODE REVIEW

**Comment by krasserm** [comment]
Thanks for the PR @odd, I'll work on the conditional Spark adapter build next week. Feel free to commit a solution should you find one earlier.

**Comment by krasserm** [comment]
Please [sign the CLA](http://rbmhtechnology.github.io/eventuate/developers.html#contributing-to-eventuate).

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.11
+sbt.version=0.13.13
```
Comment (magro) [inline project/build.properties]: As said in https://github.com/RBMHTechnology/eventuate/pull/353#issuecomment-258301692 for 0.13.13 `triggeredBy` is broken due to sbt/sbt#1444, therefore I'd suggest not to upgrade sbt here.

**Comment by magro** [comment]
It would be nice to cross-build with 2.11, WDYT?

**Comment by odd** [comment]
​Pushed new commit with downgrade of SBT to 0.13.12 and addition of cross
build for Scala 2.11.8 and 2.12.0. ​

**Comment by krasserm** [comment]
@odd please squash your commits. I'll use it as basis for further work (conditional build of Spark adapter, ...).

**Comment by krasserm** [comment]
@odd nevermind, I squashed locally and it seems to preserve the original author.

**Comment by odd** [comment]
Good, I'll be back online shortly if there is anything you want me to fix.

**Comment by krasserm** [comment]
@odd it's ok for the moment, I'll come back to you before merging to master. Thanks!

**Comment by krasserm** [comment]
Closing this PR as it is now covered by #362. @odd, I squashed your commits into a single one and made further changes in [this commit](https://github.com/RBMHTechnology/eventuate/pull/362/commits/3ec9f97e0bcd8b5299f927fd8b8894c9308cfd06). I already deployed a 0.9-SNAPSHOT version for Scala 2.12 to the snapshot repo. Please give it a try and let me know if it works. Will make a 0.8.1 release next week. Thanks again for starting the 2.12 migration.

**Pull request closed without merging**